### DISCUSSION TITLE
Send `X-Cache-Skip` header in explore mode

### DISCUSSION THREAD

**Discussion by kminehart** [grafana/grafana#38339] [Explore]
**What would you like to be added**:

We should be sending a custom header for Grafana Enterprise, `X-Cache-Skip`, in explore mode so that we do not cache requests sent from this view.

**Why is this needed**:

Caching explore mode queries is likely an unexpected behavior. Sending this header with requests will cause them to not be cached.

**Comment by marefr** [comment]
Are you planning to work on this or planning to delegate it to Observability squad?

**Comment by kminehart** [comment]
I do intend to work on it but it'd probably take me a few days to get my bearings. If anyone has a clear idea on how to do this then I would welcome it.

**Comment by marefr** [comment]
Thanks. Was just wondering whether I should delegate it to some other team. But yeah I leave this up to you then

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
